db/db_local/backup.go

```diff
@@ -160,26 +194,72 @@ func needsBackup(ctx context.Context) (bool, string, error) {
 	return false, "", nil
 }
 
-// loadBackup loads the back up file into the database
-func loadBackup(ctx context.Context) error {
-	if !helpers.FileExists(getBackupLocation()) {
+// restoreBackup loads the back up file into the database
+func restoreBackup(ctx context.Context) error {
+	if !helpers.FileExists(databaseBackupFilePath()) {
 		// nothing to do here
 		return nil
 	}
 
 	// load the db status
-	info, err := GetState()
+	runningInfo, err := GetState()
 	if err != nil {
 		return err
 	}
-	if info == nil {
+	if runningInfo == nil {
 		return fmt.Errorf("steampipe service is not running")
 	}
 
+	// get the location of the other instance which was backed up
+	_, location, err := needsBackup(ctx)
+	if err != nil {
+		return err
+	}
+
+	// extract the Table of Contents from the Backup Archive
+	toc, err := getTOCFromBackup(ctx)
+	if err != nil {
+		return err
+	}
+
+	// partition the Table of Contents into separate lists (no refresh and only refresh)
```

Comment:
make this comment longer - what is no refresh? why do we want it?

state the problem with mat views and the solution

Reply:
done

Reply:
no it's not?? Still says `// partition the Table of Contents into separate lists (no refresh and only refresh)`